cmd/sgx_plugin/README.md

```diff
@@ -101,7 +101,7 @@ the SGX DCAP out-of-tree driver is used, the `kernel.config` match expression in
 $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/operator/default?ref=<RELEASE_VERSION>
 ```
 
-**Note:** See the operator [deployment details](/cmd/operator/README.md) for its dependencies and for setting it up on systems behind proxies.
```

Comment:
Hi, thanks for the PR. But, I do not see the necessity of this change because cert manager is one of dependencies.?